hathor/p2p/manager.py

```diff
@@ -63,6 +63,8 @@ class PeerConnectionsMetrics(NamedTuple):
 class ConnectionsManager:
     """ It manages all peer-to-peer connections and events related to control messages.
     """
+    # Interval to update which peers are syncing.
+    SYNC_UPDATE_INTERVAL: int = 10 * 60  # in seconds
```

Comment:
Why not a parameter in config?

Reply:
I agree that would be good. I'm testing a node with a custom interval time and I would prefer if I could do that without carrying a change in code (which would make it harder to do that on a docker image, for example).

Reply:
I'll do this right away.

Reply:
Done.

Reply:
Done! Thanks!